CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
-# dbt_netsuite_source version.version
+# dbt_netsuite_source v0.12.0
+
+## Breaking Schema Changes
+- Added optional `fiscalcalendar` source table to support accurate fiscal year start dates (currently defaulted to calendar year). This table and related models (`stg_netsuite2__fiscal_calendar_tmp` and `stg_netsuite2__fiscal_calendar`) are disabled by default. To enable this feature:
+  - Quickstart users: enable the fiscalcalendar table in the connection schema tab.
```

Comment:
I noticed in the Fivetran schema tab this table is capitalized as FiscalCalendar. Should we update with this casing syntax or do we think this is ok?

Reply:
This should be fine since this is how the table will look in the destination. As long as the casing is the same in the downstream quickstart.yml that's what really matters.